FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +21,52 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+
+			switch (result) {
+			case let .success((data, response)):
+				do {
+					let items = try FeedImageMapper.map(data, response)
+					completion(.success(items))
+				} catch {
+					completion(.failure(Error.invalidData))
+				}
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+private class FeedImageMapper: Decodable {
```

Comment:
The `FeedImageMapper` doesn't need to conform to `Decodable` because it's not being decoded from an external representation.

```suggestion
private class FeedImageMapper {
```